components/a11y/src/overrides/skipLink.js

```diff
@@ -1,10 +1,11 @@
 /** @jsx jsx */
 
-import { jsx } from '@westpac/core';
+import { jsx, getLabel } from '@westpac/core';
 
 const SkipLink = ({ state: { href }, ...rest }) => <a href={href} {...rest} />;
 
 const skipLinkStyles = () => ({
+	label: getLabel('skiplink'),
```

Comment:
Can we camelCase multi-word components for consistency?

Reply:
yes please